src/include/duckdb/common/serializer/format_deserializer.hpp

```diff
@@ -30,22 +30,22 @@ class FormatDeserializer {
 public:
 	// Read into an existing value
 	template <typename T>
-	inline void ReadProperty(const char *tag, T &ret) {
-		SetTag(tag);
+	inline void ReadProperty(field_id_t field_id, const char *tag, T &ret) {
```

Comment:
I feel a bit weird by now, another `const` comment... 😅 But would it make sense to add `const` here to all the function parameters?